Filter/StringFilter.php

```diff
@@ -55,7 +55,7 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
      *
      * @return bool
      */
-    private function getOperator($type)
+    protected function getOperator($type)
```

Comment:
I'm not a big fan of this, because this does not concern only `LIKE`.

Why not adding methods like:

* `getTypeContainsOperator`
* `getTypeNotContainsOperator`
* `getTypeEqualOperator`

And call them from `getOperator`?

For #515, @marcoslibre could override `getTypeContainsOperator` and specify `ILIKE` on it.